src/clj/rems/landing_page.clj

```diff
@@ -2,20 +2,12 @@
   (:require [compojure.core :refer [GET defroutes]]
             [rems.context :as context]
             [rems.db.users :as users]
-            [rems.roles :refer [has-roles?]]
             [rems.util :refer [get-user-id]]
             [ring.util.response :refer [redirect]]))
 
 (defn- landing-page [req]
   (users/add-user! (get-user-id) context/*user*)
-  (let [redirect-to (get-in req [:session :redirect-to])]
-    (cond
-      redirect-to (assoc (redirect redirect-to)
-                         :session (dissoc (:session req) :redirect-to))
-      (has-roles? :owner) (redirect "/#/administration")
-      (has-roles? :approver) (redirect "/#/actions")
-      (has-roles? :reviewer) (redirect "/#/actions")
-      :else (redirect "/#/catalogue"))))
+  (redirect "/#/landing_page"))
 
 (defroutes landing-page-routes
   (GET "/landing_page" req (landing-page req)))
```

Comment:
couldn't you just redirect directly to `/#/landing_page` from the various auth methods?